### ISSUE TITLE
missing args.num_jobs in `train_allele_specific_models_command`

### ISSUE THREAD

**Issue by WesleyyC** [openvax/mhcflurry#149]
Hi,

In `train_allele_specific_models_command.py`, we are missing a `num_jobs` argument so in serial mode, the predictor is not updated correctly within`train_model()`. There used to be a `num_jobs` argument within the file, but probably get removed during refactoring.

Thanks,
Wes

**Comment by timodonnell** [comment]
Thanks for reporting this. I'm not sure I follow though - can you give more detail or a test code snippit showing the incorrect behavior you're seeing? 

There is a `--num-jobs` argument to the `mhcflurry-class1-train-allele-specific-models` command:

```
$ mhcflurry-class1-train-allele-specific-models -h
...
Local parallelism:
  --num-jobs N          Number of local processes to parallelize training
                        over. Set to 0 for serial run. Default: 0.
  --backend {tensorflow-gpu,tensorflow-cpu,tensorflow-default}
                        Keras backend. If not specified will use system
                        default.
  --gpus N              Number of GPUs to attempt to parallelize across.
                        Requires running in parallel.
...
```

**Comment by WesleyyC** [comment]
In your source code here:

https://github.com/openvax/mhcflurry/blob/c8976b0bc61f08b6dc50194b62abb4d82ef70301/mhcflurry/train_allele_specific_models_command.py#L196

The `arg.num_jobs` does not point to anything, and running the code in serial mode will cause this line to break:

https://github.com/openvax/mhcflurry/blob/c8976b0bc61f08b6dc50194b62abb4d82ef70301/mhcflurry/train_allele_specific_models_command.py#L309

**Comment by timodonnell** [comment]
Thanks @WesleyyC . This should be fixed in #153 , which should be merged soon

**Comment by WesleyyC** [comment]
Awesome, thanks!
